libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4556,8 +4584,7 @@ bool GCS_MAVLINK::try_send_message(const enum ap_message id)
     }
 
     case MSG_PLANCK_STATEINFO: {
-        CHECK_PAYLOAD_SIZE(PLANCK_STATEINFO);
-        send_planck_stateinfo();
+        //Don't do anything, as this is run in its own task
```

Comment:
Do we need to keep this in here? Could we delete `MSG_PLANCK_STATEINFO` from https://github.com/planckaero/ardupilot/blob/6d1928f9f5ed36975e793bfc3181a7ca3ec688df/libraries/GCS_MAVLink/ap_message.h#L76

Reply:
I'm a little afraid to because I think the stream macros may require it, and that switch statement may miss one.  I can try, though.

Reply:
I mean, it's fine to leave it it is if safer